lib/resValidator.js

```diff
@@ -130,3 +139,15 @@ function findOAPathMatchingPathname(pathname, OAPaths) {
   });
   return OAPath;
 }
+
+function getServerUrlUsed(openApiFullPath, servers) {
```

Comment:
nit: `openApiFullPath` -> `fullOpenApiPath` or `openApiPathWithServerUrl` is clearer to me

Reply:
`openApiPathWithServerUrl` is probably the clearest